### PR TITLE
Code has been Formatted by Prettier        GsSoC'23

### PR DESCRIPTION
**Pull request by Nikhil1017**
1) .prettierrc file and .prettierignore files have been added
2) The entire codebase has been formatted as per the prettier rules
Now the code looks so much more 'pretty' and is much more readable in code editors

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (lalitmee) [inline .prettierrc:1]: What about these rules? Do you think we can apply any of these?
```js
{
  "printWidth": 80,
  "tabWidth": 2,
  "useTabs": false,
  "semi": true,
  "singleQuote": true,
  "trailingComma": "all",
  "arrowParens": "avoid",
  "endOfLine": "auto"
}
```

Comment (Nikhil1017) [reply .prettierrc:1]: @lalitmee thanks for your rules, I have added them and formatted the code accordingly. The production build compiled without any issues

**Comment by Nikhil1017** [comment]
@PiyushKalyanpy

**Comment by Nikhil1017** [comment]
@lalitmee All conflicts resolved

**Comment by Nikhil1017** [comment]
@PiyushKalyanpy The longer the PR is delayed, more conflicts will come as the codebase get's changed. I cannot keep fixing them forever. It's been weeks.

**Comment by PiyushKalyanpy** [comment]
Hey @Nikhil1017 , these many changes in PR are not accepted, so please create a new pr which should only include prettierrc file and .prettierignore file.

**Pull request closed without merging**